src/main/java/randoop/generation/ForwardGenerator.java

```diff
@@ -614,6 +616,23 @@ private void randoopConsistencyTests(Sequence newSequence) {
     }
   }
 
+  /** The numeric types that "GRT Fuzzing" fuzzes. */
+  // TODO: Why are byte and char omitted?
```

Comment:
I am still uncertain whether these should be omitted. 

With `byte`, my primary concern is that they are sometimes treated as raw data rather than numeric values, and in such cases we may not want to fuzz them. Another potential problem with fuzzing bytes is their limited range, which increases the likelihood of overflow or underflow. This can cause the fuzzed value (bit-masked) to deviate from the specification (a Gaussian distribution centered around the original un-fuzzed input value). However, we decided to consider this as normal behavior and not to document it, as discussed earlier.

For `char`, the paper did not discuss how to fuzz it. One approach could be to use the original input char and classify it into categories such as "digit," "letter," "control character," etc. Depending on the category of the original input char, we might fuzz it by replacing it with a randomly selected character from the same category. We could choose to ignore `char` and not fuzz it. This decision is supported by the existence of GRT Constant Mining component. Consider the following example:
![image](https://github.com/randoop/randoop/assets/50536905/f3f40fec-7f00-47dd-916b-233512a218da)
I assume that the figure represents a very common way of using `char` in a Java program. GRT Constant Mining will extract these constants and populate the initial SequenceCollection so when selecting an input to test this method, we can cover each case. Here, we don't want to fuzz the `char` and cause it to no longer covering these cases.

What do you think about this?